code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -203,14 +203,13 @@
 
 			target.status = L2.status
 			target.switchcount = L2.switchcount
-			target.rigged = (obj_flags & EMAGGED ? 1 : 0)
 			target.brightness = L2.brightness
+			if(obj_flags & EMAGGED)
+				target.create_reagents(20, SEALED_CONTAINER | TRANSPARENT) //LIGHT_REAGENT_CAPACITY can't be accessed from here. Look for a not magic-number solution to this please
```

Comment:
the soultion would probably be moving the _light_defines.dm to the __DEFINES folder